yolo_engine: /eagle2/yolov3_fp16.engine
box3d_pb: /eagle2/box3d.pb
bonnet_engine: /eagle2/bonnetFP32softmax.engine
cam_frame_id: "camera"
cam_id: "/eagle2/kitti_06.avi"
do_viz: false
use_ros: false
K:
- - 7.070912000000e+02
  - 0.000000000000e+00
  - 6.018873000000e+02
- - 0.000000000000e+00
  - 7.070912000000e+02
  - 1.831104000000e+02
- - 0.000000000000e+00
  - 0.000000000000e+00
  - 1.000000000000e+00
D:
- - 0.
  - 0.
  - 0.
  - 0.
  - 0.
# for default size
#H:
#- - 1.38592897e-03
#  - 8.66205605e-03
#  - -2.34207693e+00
#- - 0.00000000e+00
#  - 1.73241121e-02
#  - -3.99454207e+00
#- - 0.00000000e+00
#  - 3.46482242e-05
#  - -5.98908415e-03
H:
- - 3.38645324e-03
  - 1.41102218e-02
  - -1.93022732e+00
- - -0.00000000e+00
  - 4.12018477e-02
  - -4.15071354e+00
- - -0.00000000e+00
  - 5.64408873e-05
  - -4.31604594e-03
